bin/db_login.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+
+from jobbergateapi2.config import settings
+
+
+def login():
+    subprocess.run(["pgcli", settings.DATABASE_URL])
```

Comment:
What is this file for?

Reply:
This is a convenience script for logging into the database that the project is set up to use. Once you have your configuration set up, you can simply run:
```
$ poetry run db-login
```

And you will be dropped into an interactive database shell.

It saves you from having to remember and type in the full postgres command every time.